### PR TITLE
Manolo tlc arc day 9 9_4 to 9_4B

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Arcueid/Day 9/09_00_ARC09_4.txt [hunk 1]
```diff
@@ -71,7 +72,7 @@ I don't remember much about it at the very least.
 [sha:0607cfa8f1d558e981c5e807d1037385baa8a4a8]{
 -- Page 127, Offset 21632.
 -- 森の外からでは決して見えない死角で、屋敷にいる時は……翡翠に付いてこなければ一生気づかなかったぐらいの、木々に囲まれた小さな広場。
-In a blind spot that could never been from outside the forest, there is a small field surrounded by trees... If I hadn't followed Hisui, I may never have found this in my entire life.
+In a blind spot that could never be seen from outside the forest, there's a small field surrounded by trees... If I hadn't followed Hisui, I may have never found this place.
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_4.txt:75]: In a spot invisible to the outside world, there's a small clearing in the forest... If I hadn't followed Hisui, I may have found this place hidden by the trees.

File: script/Arcueid/Day 9/09_00_ARC09_4.txt [hunk 2]
```diff
@@ -246,7 +247,7 @@ He's dead. He's dead. He's dead.
 [sha:d38c1e89802894e0d55dc597034945649c8ab523]{
 -- Page 135, Offset 21667.
 -- ――――どこかで、<蝉|ヒト>の声がしている。
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_4.txt:249]: this is just cruel to translators

File: script/Arcueid/Day 9/09_00_ARC09_4.txt [hunk 3]
```diff
@@ -291,7 +292,8 @@ My wound opens.
 [sha:7e80057d5d0005c3c2664ba40a3a3bf0eb865cef]{
 -- Page 137, Offset 21676.
 -- 胸が真っ赤に染まって、この両手まで赤黒く<赫灼|かくしゃく>と。
-My chest dyes in deep red, and even my hands glistens in a reddish brown.
+// could possibly use some other shades of red for stylistic effect here...?
+My chest is dyed in a deep red, and even my hands glisten in a brownish-red.
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_4.txt:296]: glisten in crimson

File: script/Arcueid/Day 9/09_00_ARC09_4.txt [hunk 4]
```diff
@@ -331,6 +334,7 @@ I see the tear-stained face of Akiha covered in blood.
 [sha:8bf2b2be5c504a787946f1863068bbdecfca0df9]{
 -- Page 139, Offset 21684.
 -- ……嗚呼。
+// 嗚呼 the more you know i guess
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_4.txt:337]: Japanese moment

File: script/Arcueid/Day 9/09_00_ARC09_4B.txt [hunk 5]
```diff
@@ -11,7 +11,8 @@
 [sha:55afaa7c0e935c7d70182255cb3c427d0751e94c]{
 -- Page 194, Offset 21846.
 -- 「馬鹿なことは言わないで翡翠。呼べる訳がないでしょう、兄さんの傷は普通の傷じゃないんだから……！」
-"Don't say something so outrageous, Hisui. There's no reason to call them, is there? Nii-san's wound isn't normal after all...!"
+// i think this line would benefit from a healthy serving of expletives
+"Don't say such idiotic things, Hisui. There's no reason to call one, is there? Nii-san's wound isn't normal after all...!"
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_4B.txt:15]: revert this imo "馬鹿なことは言わない" as "don't say such idiotic things" is an antipattern

File: script/Arcueid/Day 9/09_00_ARC09_4B.txt [hunk 6]
```diff
@@ -191,7 +194,7 @@ This is the best I can do for now. I don't think I can even move a single finger
 [sha:3762739dc81d3998fe60ce718c92579faede2f6c]{
 -- Page 209, Offset 21882.
 -- けど翡翠が落ちこんでいる手前、無理をしてでも元気なフリをしなくちゃいけない。
-But since Hisui is feeling down, I have to pretend that I'm feeling well even if I have to force myself.
+But since Hisui's feeling down, I have to pretend that I'm fine, even if I have to force myself.
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_4B.txt:197]: bring back "is" here

File: script/Arcueid/Day 9/09_00_ARC09_4_2.txt [hunk 7]
```diff
@@ -191,7 +191,7 @@ I was able to walk, but it took me around another hour to arrive due to my snail
 [sha:cc16d29c845937bb942d8e5031829bd4ac5243e0]{
 -- Page 249, Offset 21986.
 -- 肩で呼吸をしながら周囲を見渡す。
-I breathe up to my shoulders as I look around my surroundings.
+I look around my surroundings while breathing heavily.
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_4_2.txt:194]: surroundings, breathing heavily.

File: script/Arcueid/Day 9/09_00_ARC09_4_2.txt [hunk 8]
```diff
@@ -441,12 +443,12 @@ Before I could pull my body away, Arcueid's chest comes into contact with mine.
...
 -- Page 268, Offset 22036.
 -- いつもの眩暈とは違った立ち眩みが、脳内に駆け巡る。
-Although it's different from the usual dizziness, it runs about through the insides of my brain.
+Although it's different from the usual dizziness, it runs around the insides of my brain.
 }
 [sha:a6558b9f71e2a2a64b4ebe3f997234dad0508fe0]{
 -- Page 268, Offset 22037.
 -- 今まで意識しないよう努めていた、アルクェイドの胸の柔らかさとか、肌の白さとか、キレイな金の髪とか、そういった女性的な艶めかしさが。
-I tried my best to become unaware of it up until now, whether it be the softness of her breasts, her pure white skin, her pretty golden hair, or her feminine charm.
+I tried my best to remain unaware of it up until now, whether it be the softness of her breasts, her pure white skin, her pretty golden hair, or her feminine charm.
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_4_2.txt:451]: shiki moment

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 9/09_00_ARC09_4.txt (hunk 2 above)
Comment (nolo16) [reply script/Arcueid/Day 9/09_00_ARC09_4.txt:249]: :Saoko:

**Pull request merged**